packages/editor/src/plugins/image/components/licence-dropdown.tsx

```diff
@@ -0,0 +1,111 @@
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import { faQuestionCircle } from '@fortawesome/free-regular-svg-icons'
+import { faCaretDown, faCaretUp } from '@fortawesome/free-solid-svg-icons'
+import * as Select from '@radix-ui/react-select'
+import React, { useState, useEffect, useRef } from 'react'
+
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { cn } from '@/helper/cn'
+
+interface LicenseDropdownProps {
+  defaultLicense?: string
+  onLicenseChange?: (license: string) => void
+  isPixabayImage?: boolean
+}
+
+export const LicenseDropdown: React.FC<LicenseDropdownProps> = ({
+  onLicenseChange,
+  isPixabayImage = false,
+}) => {
+  const editorStrings = useEditorStrings()
+  const imageStrings = editorStrings.plugins.image
+  const { licences, licenceHelpText } = imageStrings
+  const licenceNames = Object.values(licences)
+
+  const [selectedLicense, setSelectedLicense] = useState(licenceNames[0])
+  const [isConfirmed, setIsConfirmed] = useState(false)
+  const [isOpen, setIsOpen] = useState(false)
+  const [triggerWidth, setTriggerWidth] = useState(0)
```

Comment:
@hejtful @elbotho I didn't find a better way to get the content of the dropdown and the select box itself to keep the same width. If you guys are unhappy with this implementation I would be grateful for a short pairing session to check it out together

Reply:
I would not mind if box and dropdown don't have the same width and would probably not add the complexity for a select box in a settings modal

Reply:
I tend to agree but since the focus of the ticket is about redesign so it seems that design and UX is important and I wouldn't feel comfortable having it not match the design exactly without having input from Gregor

Reply:
>I wouldn't feel comfortable having it not match the design exactly without having input from Gregor

The question mark icon isn't right-aligned, and it's right aligned in the design, so it's not matching the design exactly anyway.

Plus, on Safari, there's this tiny hole in the border.
<img width="172" alt="Screenshot 2024-07-03 at 16 57 45" src="https://github.com/serlo/frontend/assets/13198821/cc06f312-333a-445c-ac70-fdc6b3eb7766">

My point is, no need to go for pixel perfect, if something increases complexity in the code and doesn't bring a lot of value in the UX, it's good to avoid the complexity. If you do believe that this brings value to UX however, then there should be an editor-wide dropdown component detached from the licencing logic, where this UI fix would belong.

Reply:
I will try to make a seperate dropdown component as you suggest

Reply:
@hejtful I improved the component and made it an shared editor-wide dropdown component, it seems to work well now.